source/world_builder/features/fault.cc

```diff
@@ -51,6 +51,39 @@ namespace WorldBuilder
       = default;
 
 
+
+    void Fault::make_snippet(Parameters &prm)
+    {
+      using namespace rapidjson;
+      Document &declarations = prm.declarations;
+
+      const std::string path = prm.get_full_json_path();
+
+      /*
+      ideally:
+       {
+        "model": "fault",
+        "name": "${1:default name}",
+        "dip point":[0.0,0.0],
+        "coordinates": [[0.0,0.0]],
+        "segments": [],
+        "sections": [],
+        "temperature models":[{"model":"uniform", "temperature":600.0}],
+        "composition models":[{"model":"uniform", "compositions": [0], "fractions":[1.0]}]
+       }
+       */
+
+      Pointer((path + "/body").c_str()).Set(declarations,"object");
+      Pointer((path + "/body/model").c_str()).Set(declarations,"fault");
+      Pointer((path + "/body/name").c_str()).Set(declarations,"${1:Menno's Fault}");
```

Comment:
I am not contradicting the default name here ;) Still, it think it might be better to align all the default names. I like the `My ...` version, because it indicates that you can write something there, more then the lower case `mantle layer` for example.